README.md

```diff
@@ -10,8 +10,15 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10797541.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
 
-**TrixiParticles.jl** is a numerical simulation framework designed for particle-based numerical methods, with an emphasis on multiphysics applications, written in [Julia](https://julialang.org).
-A primary goal of the framework is to be user-friendly for engineering, science, and educational purposes. In addition to its extensible design and optimized implementation, we prioritize the user experience, including installation, pre- and postprocessing.
+**TrixiParticles.jl** is a high-performance numerical simulation framework for particle-based methods, focused on the simulation of complex multiphysics problems, and written in [Julia](https://julialang.org).
+
+TrixiParticles.jl focuses on the following use cases:
+- Development of new particle-based methods and models.
+- Accurate and efficient physics-based modelling of complex multiphysics problems.
+- Easy setup of accessible simulations for educational purposes, including student projects, coursework, and thesis work.
+
+It offers intuitive configuration, robust pre- and post-processing, and vendor-agnostic GPU-support based on the Julia package [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl). 
+In the future we plan to enable large-scale simulations through adding MPI support.
```

Comment:
```suggestion
```
I wouldn't put this here until it's actually available.

Reply:
I mostly added this since the question regarding "need" came up in the JOSS review. But we can also leave it out for now.

Reply:
Usually I am not a big fan of "announcing" planned features, but rather just have them and then report on them :-)